packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -83,6 +85,23 @@ export const PostCard = forwardRef(function PostCard(
       ),
     [postCardVersion],
   );
+
+  const getClickablePost = () => {
```

Comment:
It doesn't have to be a function. This can be a variable that stores the right elements